### DISCUSSION TITLE
Toast: Opacity

### DISCUSSION THREAD

**Discussion by fotisskal** [primefaces/community#932] [PrimeVue]
The default **Toast** message, when displayed, is transparent, as a result it overlaps with the topbar. Is there any option to make it opaque? I changed the Toast element's (in AppLayout) opacity to 1 with scoped style, but nothing has changed. Thank you in advance.

**Comment by mertsincan** [comment]
@tugcekucukoglu

**Reply by fotisskal** [reply]
hello @mertsincan and @tugcekucukoglu , any update on this?

**Comment by fotisskal** [comment]
A possible workaround to this is to change the background color of the content to something darker eg.

`toast.add({ severity: "success", summary: "Successful", detail: "....", contentStyleClass: "bg-gray-800", life: 3000 });`
